test/rest.test.js

```diff
@@ -210,6 +210,24 @@ describe('strong-remoting-rest', function() {
         .end(done);
     });
 
+    it('should use `content-type: application/json` for default errorHandler',
```

Comment:
Could you please add a comment before this test case with a link to the github issue this test is reproducing?